packages/wrangler/src/dev.ts

```diff
@@ -324,6 +324,12 @@ export const dev = createCommand({
 				"Bind to production Vectorize indexes in local development mode",
 			default: false,
 		},
+		"experimental-images-local-mode": {
+			type: "boolean",
+			describe:
+				"Use a local lower-fidelity implementation of the Images binding",
+			default: false,
+		},
```

Comment:
I know this is a product call, but I would really suggest flipping the default here. Local-mode development in Wrangler is the norm, and I think having remote be an opt-in would fit in with that better

Reply:
The local mode is really missing a lot of features compared to remote, so I don't think we would really want it to be the default. I suppose nothing really prevents us from changing that down the line though, vs switching the default from local to remote.